### PR TITLE
Indigo devel

### PR DESCRIPTION
**Pull request by yinxchun**

### CODE REVIEW

**Comment by davetcoleman** [comment]
@mlautman

File: conf.py [hunk 1]
```diff
@@ -38,7 +38,7 @@
     "display_github": True,
     "github_user": "ros-planning",
     "github_repo": "moveit_tutorials",
-    "github_version": "master/",
+    "github_version": "indigo-devel/",
```
Comment (mlautman) [inline conf.py]: kinetic-devel

File: conf.py [hunk 2]
```diff
@@ -53,7 +53,7 @@
 # Add any paths that contain custom themes here, relative to this directory.
 
 # Links
-extlinks = {'codedir': ('https://github.com/ros-planning/moveit_tutorials/tree/master/doc/pr2_tutorials//%s', ''),
+extlinks = {'codedir': ('https://github.com/ros-planning/moveit_tutorials/tree/indigo-devel/doc/pr2_tutorials//%s', ''),
```
Comment (mlautman) [inline conf.py]: Same as above. Also, you can just reference the values from the html_context dict like I did here
https://github.com/PickNikRobotics/moveit_tutorials/blob/7dcb56d657e6c7150fe7a557c28762f463e4040b/conf.py#L56

File: doc/ikfast_tutorial.rst [hunk 3]
```diff
@@ -26,13 +26,15 @@ Install the MoveIt! IKFast package either from debs or from source.
...
+ sudo apt-get install ros-indigo-moveit-kinematics
 
 **Source**
 
 Inside your catkin workspace ::
 
- git clone https://github.com/ros-planning/moveit_ikfast.git
+ git clone https://github.com/ros-planning/moveit.git
+ cd moveit
+ git checkout indigo-devel
```
Comment (mlautman) [inline doc/ikfast_tutorial.rst]: kinetic-devel

File: doc/pr2_tutorials/planning/scripts/doc/move_group_python_interface_tutorial.rst [hunk 4]
```diff
@@ -16,7 +16,7 @@ The entire code can be seen :codedir:`here in the moveit_pr2 github project<plan
 The launch file
 ^^^^^^^^^^^^^^^
 The entire launch file is `here
-<https://github.com/ros-planning/moveit_tutorials/tree/master/doc/pr2_tutorials/planning/launch/move_group_python_interface_tutorial.launch>`_
+<https://github.com/ros-planning/moveit_tutorials/tree/indigo-devel/doc/pr2_tutorials/planning/launch/move_group_python_interface_tutorial.launch>`_
```
Comment (mlautman) [inline doc/pr2_tutorials/planning/scripts/doc/move_group_python_interface_tutorial.rst]: reference ext_links in conf.py

Comment (mlautman) [reply doc/pr2_tutorials/planning/scripts/doc/move_group_python_interface_tutorial.rst]: eg :codedir:`here<move_group_python_interface/launch/move_group_python_interface_tutorial.launch>`

File: doc/pr2_tutorials/planning/src/doc/controller_configuration.rst [hunk 5]
```diff
@@ -96,3 +96,14 @@ The ``FollowJointTrajectory`` or ``GripperCommand`` interfaces on your robot mus
  * /gripper_controller/gripper_action/result
 
 You should also be able to see (using *rostopic info topic_name*) that the topics are published/subscribed to by the controllers on your robot and also by the *move_group* node.
+
+Remapping /joint_states topic
+-----------------------------
+
+When you run a `move group node <http://docs.ros.org/indigo/api/moveit_tutorials/html/doc/pr2_tutorials/planning/src/doc/move_group_interface_tutorial.html>`_, you may need to remap the topic /joint_states to /robot/joint_states, otherwise MoveIt! won't have feedback from the joints. To do this remapping you could make a simple launch file for your node as follows ::
```
Comment (mlautman) [inline doc/pr2_tutorials/planning/src/doc/controller_configuration.rst]: reference conf.py variables.

File: doc/pr2_tutorials/planning/src/doc/motion_planning_api_tutorial.rst [hunk 6]
```diff
@@ -17,7 +17,7 @@ Follow the `instructions for compiling code from source <http://moveit.ros.org/i
 
 The launch file
 ^^^^^^^^^^^^^^^
-The entire launch file is `here <https://github.com/ros-planning/moveit_tutorials/tree/master/doc/pr2_tutorials/planning/launch/planning_pipeline_tutorial.launch>`_ on github. All the code in this tutorial can be compiled and run from the moveit_tutorials package
+The entire launch file is `here <https://github.com/ros-planning/moveit_tutorials/tree/indigo-devel/doc/pr2_tutorials/planning/launch/planning_pipeline_tutorial.launch>`_ on github. All the code in this tutorial can be compiled and run from the moveit_tutorials package
```
Comment (mlautman) [inline doc/pr2_tutorials/planning/src/doc/motion_planning_api_tutorial.rst]: conf.py

File: doc/pr2_tutorials/planning/src/doc/move_group_interface_tutorial.rst [hunk 7]
```diff
@@ -15,7 +15,7 @@ Follow the `instructions for compiling code from source <http://moveit.ros.org/i
 
 The launch file
 ^^^^^^^^^^^^^^^
-The entire launch file is `here <https://github.com/ros-planning/moveit_tutorials/tree/master/doc/pr2_tutorials/planning/launch/move_group_interface_tutorial.launch>`_ on github. All the code in this tutorial can be compiled and run from the moveit_tutorials package
+The entire launch file is `here <https://github.com/ros-planning/moveit_tutorials/tree/indigo-devel/doc/pr2_tutorials/planning/launch/move_group_interface_tutorial.launch>`_ on github. All the code in this tutorial can be compiled and run from the moveit_tutorials package
```
Comment (mlautman) [inline doc/pr2_tutorials/planning/src/doc/move_group_interface_tutorial.rst]: conf.py

File: doc/pr2_tutorials/planning/src/doc/planning_pipeline_tutorial.rst [hunk 8]
```diff
@@ -21,7 +21,7 @@ Follow the `instructions for compiling code from source <http://moveit.ros.org/i
 
 The launch file
 ^^^^^^^^^^^^^^^
-The entire launch file is `here <https://github.com/ros-planning/moveit_tutorials/tree/master/doc/pr2_tutorials/planning/launch/planning_pipeline_tutorial.launch>`_ on github. All the code in this tutorial can be compiled and run from the moveit_tutorials package that you have as part of your MoveIt! setup.
+The entire launch file is `here <https://github.com/ros-planning/moveit_tutorials/tree/indigo-devel/doc/pr2_tutorials/planning/launch/planning_pipeline_tutorial.launch>`_ on github. All the code in this tutorial can be compiled and run from the moveit_tutorials package that you have as part of your MoveIt! setup.
```
Comment (mlautman) [inline doc/pr2_tutorials/planning/src/doc/planning_pipeline_tutorial.rst]: conf.py

File: doc/pr2_tutorials/planning/src/doc/planning_scene_ros_api_tutorial.rst [hunk 9]
```diff
@@ -19,7 +19,7 @@ Follow the `instructions for compiling code from source <http://moveit.ros.org/i
 
 The launch file
 ^^^^^^^^^^^^^^^
-The entire launch file is `here <https://github.com/ros-planning/moveit_tutorials/tree/master/doc/pr2_tutorials/planning/launch/planning_scene_ros_api_tutorial.launch>`_ on github. All the code in this tutorial can be compiled and run from the moveit_tutorials package
+The entire launch file is `here <https://github.com/ros-planning/moveit_tutorials/tree/indigo-devel/doc/pr2_tutorials/planning/launch/planning_scene_ros_api_tutorial.launch>`_ on github. All the code in this tutorial can be compiled and run from the moveit_tutorials package
```
Comment (mlautman) [inline doc/pr2_tutorials/planning/src/doc/planning_scene_ros_api_tutorial.rst]: conf.py

File: doc/pr2_tutorials/planning/src/doc/planning_scene_tutorial.rst [hunk 10]
```diff
@@ -17,7 +17,7 @@ Follow the `instructions for compiling code from source <http://moveit.ros.org/i
 
 The launch file
 ^^^^^^^^^^^^^^^
-The entire launch file is `here <https://github.com/ros-planning/moveit_tutorials/tree/master/doc/pr2_tutorials/planning/launch/planning_scene_tutorial.launch>`_ on github. All the code in this
+The entire launch file is `here <https://github.com/ros-planning/moveit_tutorials/tree/indigo-devel/doc/pr2_tutorials/planning/launch/planning_scene_tutorial.launch>`_ on github. All the code in this
```
Comment (mlautman) [inline doc/pr2_tutorials/planning/src/doc/planning_scene_tutorial.rst]: conf.py

File: index.rst [hunk 11]
```diff
@@ -5,6 +5,8 @@ These tutorials will run you through how to use MoveIt! with your robot. It is a
 
 .. note:: All tutorials referencing the PR2 have only been tested with ROS Indigo but likely work with ROS Jade. See `issue <https://github.com/ros-planning/moveit/issues/110>`_ for more information.
 
+Before beginning the tutorials it is recommended that you read through the `Concepts <http://moveit.ros.org/documentation/concepts/>`_ page to get a general overview of how MoveIt! works. This page runs through some of the basic concepts underlying the MoveIt! system architecture, interfaces and usage.
```
Comment (mlautman) [inline index.rst]: conf.py

**Comment by mlautman** [comment]
Please include a description with your PR explaining the intention so we can best help with the merge process.

**Pull request closed without merging**